pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -438,6 +446,130 @@ func (o *InstallOptions) Run(ctx context.Context) error {
 	return nil
 }
 
+func (o *InstallOptions) waitForOtherInstallerRevisionsToSettle(ctx context.Context) error {
+	currRevision64, err := strconv.ParseInt(o.Revision, 10, 32)
+	if err != nil {
+		return fmt.Errorf("bad local revision %v: %w", o.Revision, err)
+	}
+	currRevision := int(currRevision64)
+
+	// at this point we have a list of all the installer pods present on this node we need to check
+	// 1. is there a revision later than us present right now.  If yes, exit.
+	// 2. is there an installer pod present that has not finished running, wait until it is finished.
+	err = wait.PollImmediateWithContext(ctx, 10*time.Second, 60*time.Second, func(ctx context.Context) (done bool, err error) {
+		installerPods, err := o.getInstallerPodsOnThisNode(ctx)
+		if err != nil {
+			return false, nil
+		}
+		if len(installerPods) == 0 {
+			return false, fmt.Errorf("no installer pods found")
+		}
+		latestRevision, err := getRevisionOfPod(installerPods[len(installerPods)-1])
+		if err != nil {
+			return false, err
+		}
+		if latestRevision > currRevision {
+			return false, fmt.Errorf("more recent revision present on node: thisRevision=%v, moreRecentRevision=%v", currRevision, latestRevision)
+		}
+
+		// if any container is not terminated, we need to wait.  There should only ever be a single installer pod
+		// that could be run
+		for _, pod := range installerPods {
+			// skip this revision
+			podRevision, err := getRevisionOfPod(pod)
+			if err != nil {
+				return false, err
+			}
+			if podRevision == currRevision {
+				continue
+			}
+			for _, container := range pod.Status.ContainerStatuses {
+				if container.State.Terminated == nil {
```

Comment:
To related the code changes to https://bugzilla.redhat.com/show_bug.cgi?id=2028595#c1:
```
08:50:14.597891 - installer-6-ip-10-0-129-204.us-east-2.compute.internal created
08:50:14.915568 - installer-6-ip-10-0-129-204.us-east-2.compute.internal waiting for its sandbox
08:50:20.195360 - installer-6-ip-10-0-129-204.us-east-2.compute.internal requested to be deleted
08:50:24.001152 - installer-7-ip-10-0-129-204.us-east-2.compute.internal created
08:50:28.014171 - installer-7-ip-10-0-129-204.us-east-2.compute.internal waiting for its sandbox
08:50:25.997697 - installer-7-ip-10-0-129-204.us-east-2.compute.internal installer container started to run
08:50:27.013251 - installer-7-ip-10-0-129-204.us-east-2.compute.internal installer container stopped running/got deleted
08:50:47.250319 - installer-6-ip-10-0-129-204.us-east-2.compute.internal installer container started to run
08:50:48.265838 - installer-6-ip-10-0-129-204.us-east-2.compute.internal installer container stopped running
```

it may take at least 34s from creating an installer pod until it stops running. In this case the installer rev-7 will have to wait for installer rev-6 for 24s.

Reply:
> it may take at least 34s from creating an installer pod until it stops running. In this case the installer rev-7 will have to wait for installer rev-6 for 24s.

for each installer pod that is created, that is true. If this fails, we will create another install pod.  Waiting for 24s looks correct, I made the wait 60s as I recall.